Makefile

```diff
@@ -141,6 +141,10 @@ endif
 docker-otelcontribcol:
 	COMPONENT=otelcontribcol $(MAKE) docker-component
 
+.PHONY: generate
+generate:
+	$(MAKE) -j4 for-all CMD="go generate ./..."
```

Comment:
I don't think the -j4 works with for-all. I think you have to use for-all-target IIRC.